packages/hurumap-core/src/Scope/utils.js

```diff
@@ -0,0 +1,28 @@
+function idify(string) {
+  return string
+    .replace(/^\s+|\s+$/g, "")
+    .replace(/[^a-z0-9]/g, "")
+    .replace(/\s+/g, "_")
+    .replace(/_+/g, "_");
+}
```

Comment:
```suggestion
function idify(string) {
  return string
      .toLowerCase()
    .replace(/^\s+|\s+$/g, "")
    .replace(/[^a-z0-9]/g, "")
    .replace(/\s+/g, "_")
    .replace(/_+/g, "_");
}
```
Just to make sure we don't lose anything that comes with uppercase